src/java.base/share/classes/sun/net/www/protocol/http/DigestAuthentication.java

```diff
@@ -513,7 +513,7 @@ private String getHeaderValueImpl (String uri, String method) {
         String ncstring=null;
 
         if (nccount != -1) {
-            ncstring = Integer.toHexString (nccount).toLowerCase();
+            ncstring = Integer.toHexString (nccount).toLowerCase(Locale.ROOT);
```

Comment:
```suggestion
            ncstring = Integer.toHexString(nccount);
```
`toHexString` returns lowercase string